app/src/main/java/org/mozilla/fenix/home/recenttabs/view/RecentTabViewHolder.kt

```diff
@@ -37,6 +40,10 @@ class RecentTabViewHolder(
         composeView.setContent {
             val recentTabs = store.observeAsComposableState { state -> state.recentTabs }
 
+            if (!recentTabs.value.isNullOrEmpty()) {
+                components.analytics.experiments.recordExposureEvent(FeatureId.SEARCH_TERM_GROUPS)
+            }
+
```

Comment:
Same as https://github.com/mozilla-mobile/fenix/pull/22481#discussion_r763355526

Reply:
If the 'recentTabs' is null or empty for the control branch, we still want to record an "exposure" as it indicates the point at which a user _would have_ seen the treatment if there was one.

Reply:
That totally makes sense. Thanks for the clarification!